website/content/commands/acl/token/update.mdx

```diff
@@ -49,9 +52,16 @@ Usage: `consul acl token update [options]`
   be specified multiple times. Format is `NODENAME:DATACENTER`. Added in Consul
   1.8.1.
 
-- `-policy-id=<value>` - ID of a policy to use for this token. May be specified multiple times.
+- `-policy-id=<value>` - ID of a policy to use for this token. Overwrites existing policies. May be specified multiple times.
+
+- `-policy-name=<value>` - Name of a policy to use for this token. Overwrites existing policies. May be specified multiple times.
+
+~> `-policy-id` and `-policy-name` will completely overwrite existing policies. Use `-add-policy-id` or `-add-policy-name` if you
```

Comment:
I'm not sure about the convention for adding notes inside command descriptions in the API docs. Are there other examples where we do this? I'm just curious.

Reply:
an example is here: https://developer.hashicorp.com/consul/commands/acl/token/update#usage 

<img width="742" alt="Screen Shot 2023-02-17 at 9 08 29 AM" src="https://user-images.githubusercontent.com/7438896/219677604-0ff7a5a2-aabb-42fe-bbec-c7491dda6d47.png">